### DISCUSSION TITLE
ELO Form

### DISCUSSION THREAD

**Discussion by lishaduck** [PSDTools/.github#9] [General]
Posting this again, because you still haven't replied, @ParkerH27 / @PetalCat:

> What is the title? I'm filling out the form, and we need a name for our project. Also, we have committed to Flutter, right? Do you have anyone in mind for our mentors (you know this, I think, but we need one from in district, and [optionally] someone out-of-district)? Is this a good [timeline](https://www.icloud.com/freeform/00eaCA9rLgi-rgcUX_DJ-rpyQ#Wallet)? Finally, one of us needs to keep our Macs over the summer per the [Flutter docs](https://docs.flutter.dev/get-started/install/macos#macos-setup).

> Also, you had a hard time drawing charts before: try [mermaidjs](https://mermaid.live/edit).

Answers so far:
- Flutter: yes
- Mentors: Carson?
- Timeline: see psdtools/app#8
- Macs: you'll keep one for summer school

Unanswered:
- Name
- Out-of-school mentors
- Approval of
  - psdtools/app#8
  - Carson for our mentor

## Misc
- I've also begun a C4 chart via mermaidjs on the [readme](https://github.com/PSDTools/app#c4-diagram).
- #8

**Comment by ParkerH27** [comment]
I think the name can be Pirate Wallet or something similar
I don't know about a mentor
And macs: I will not be taking summer school but I can get a mac.

**Comment by lishaduck** [comment]
Thanks!

**Comment by lishaduck** [comment]
FYI, I have a candidate for a career mentor. Ask me in class.

**Reply by PetalCat** [reply]
Awesome! Will do!
